src/plugins/multiline/index.ts

```diff
@@ -160,6 +176,53 @@ export default class Multiline extends PluginController<Config> {
       const remove = hookIntoOverrideKeystroke(
         Calc.focusedMathQuill.mq,
         (key, _) => {
+          const mq = Calc.focusedMathQuill?.mq;
+
+          if (key === "Shift-Enter" && this.settings.spacesToNewlines) {
+            if (mq) {
+              mq.typedText("   ");
+              this.enqueueVerticalifyOperation(
+                mq.__controller.container.querySelector(".dcg-mq-root-block")!
+              );
+              setTimeout(() => {
+                this.dequeueAllMultilinifications();
+              });
+            }
+
+            return false;
+          }
+
+          if (mq && key === "Backspace" && this.settings.spacesToNewlines) {
+            if (isNextToTripleSpaceLineBreak(mq, -1)) {
```

Comment:
Remark: Mathquill lets `/** left */ const L = -1` and `/** right */ const R = +1`, so there's not just magical -1 and +1 everywhere.

Reply:
Added.